sormas-backend/src/main/java/de/symeda/sormas/backend/task/TaskFacadeEjb.java

```diff
@@ -700,7 +703,12 @@ public void sendNewAndDueTaskMessages() {
 						: String.format(
 							I18nProperties.getString(MessagingService.CONTENT_TASK_START_SPECIFIC),
 							task.getTaskType().toString(),
-							context.toString() + " " + DataHelper.getShortUuid(associatedEntity.getUuid()));
+							context.toString() + (associatedEntity != null ? (" " + DataHelper.getShortUuid(associatedEntity.getUuid())) : ""));
+
+					String associatedEntityLinkMessagePart = getAssociatedEntityLinkMessagePart(context, associatedEntity);
```

Comment:
extract these two additions (this and on line 744) that are the same to a method, for eg. appendAssociatedEntityLink(...)